config/tests/config_tests.rs

```diff
@@ -29,6 +29,24 @@ use std::{fs::File, io::Write};
 use tempfile::tempdir;
 use test_utils::make_authority_with_port_getter;
 
+#[test]
+fn leader_election_rotates_through_all() {
+    // this committee has equi-sized stakes
+    let committee = test_utils::committee(None);
+    let mut leader_counts = HashMap::new();
+    let mut leader_counts_stepping_by_2 = HashMap::new();
+    for i in 0..100 {
+        let leader = committee.leader(i);
+        let leader_stepping_by_2 = committee.leader(i * 2);
```

Comment:
It's not immediately clear to me what we are trying to test by having the `leader_stepping_by_2` here. @huitseeker could you please provide some info?

Reply:
I've added a comment: we iterate through even rounds in practice, and so I wanted to make sure we hit the whole roster (each available validator) reliably.